confgenerator/logging.go

```diff
@@ -21,12 +21,15 @@ import (
 )
 
 // setLogNameComponents generates a series of components that rewrites the tag on log entries tagged `tag` to be `logName`.
-func setLogNameComponents(tag, logName, receiverType string) []fluentbit.Component {
+func setLogNameComponents(tag, logName, receiverType string, hostName string) []fluentbit.Component {
 	return LoggingProcessorModifyFields{
 		Fields: map[string]*ModifyField{
 			"logName": {
 				DefaultValue: &logName,
 			},
+			`labels."agent.googleapis.com/resource_name"`: {
```

Comment:
I think from reading the tea leaves (https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/dce3e4b53ec4ff35e72d3c5fac1cf38428048e9d/lib/fluent/plugin/common.rb#L23 and https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/dce3e4b53ec4ff35e72d3c5fac1cf38428048e9d/lib/fluent/plugin/out_google_cloud.rb#L1126) that this should actually be

```
compute.googleapis.com/resource_name
```

to match the legacy logging agent.

Reply:
That's right! Fixed.